.scratch-16119/ablate.py

```
@@ -0,0 +1,295 @@
+            ok=False
+            print(f"{aid}: ANCHOR MISS ({s.count(old)}) in {path}", flush=True)
+            break
+        s = s.replace(old,new)
+    if not ok:
+        results.append((aid,desc,"ANCHOR-MISS",""))
+        continue
+    io.open(path,'w',encoding='utf-8').write(s)
+    try:
+        code, out = run_tests(tests)
+    finally:
+        subprocess.run(["git","checkout","--","path" if False else path], cwd=REPO)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)